# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/html/HTMLSelectAccessible.h
# Commit: 1685016c0c79
# Full Hash: 1685016c0c79535b5cc70c0a33d7bab682e90a18
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-11-11 21:17:02
# Regressor Bug: 1733263
# File Overlap Count: 1
# Description:
#   Bug 1733263 - P5: Rely on DOMMenuItem events for ACTIVE state changes in select elements. r=Jamie
#   
#   1. Use dom events in RootAccessible to fire ACTIVE state changes.
#   2. Add DOMMenuItemInactive events to nsListControlFrame and fire it on
#      the previously active option.
# ==============================================================================

diff -r edcc0183f30d -r 1685016c0c79 accessible/html/HTMLSelectAccessible.h
--- a/accessible/html/HTMLSelectAccessible.h	Thu Nov 11 17:05:58 2021 +0000
+++ b/accessible/html/HTMLSelectAccessible.h	Thu Nov 11 17:05:58 2021 +0000
@@ -71,6 +71,8 @@
   virtual nsRect RelativeBounds(nsIFrame** aBoundingFrame) const override;
   virtual void SetSelected(bool aSelect) override;
 
+  nsresult HandleAccEvent(AccEvent* aEvent) override;
+
   // ActionAccessible
   virtual uint8_t ActionCount() const override;
   virtual void ActionNameAt(uint8_t aIndex, nsAString& aName) override;